tests/cli/ursula/test_blockchain_ursula.py

```diff
@@ -0,0 +1,359 @@
+import json
+import os
+import random
+import secrets
+import string
+
+import datetime
+import maya
+import pytest
+from umbral.keys import UmbralPublicKey
+
+from nucypher.blockchain.eth.actors import Miner
+from nucypher.blockchain.eth.agents import NucypherTokenAgent, MinerAgent
+from nucypher.blockchain.eth.constants import MIN_LOCKED_PERIODS, MIN_ALLOWED_LOCKED
+from nucypher.characters.lawful import Enrico
+from nucypher.cli.main import nucypher_cli
+from nucypher.config.characters import UrsulaConfiguration, BobConfiguration
+from nucypher.utilities.sandbox import constants
+from nucypher.utilities.sandbox.blockchain import token_airdrop
+from nucypher.utilities.sandbox.constants import (
+    MOCK_IP_ADDRESS,
+    TEST_PROVIDER_URI,
+    MOCK_URSULA_STARTING_PORT,
+    INSECURE_DEVELOPMENT_PASSWORD,
+    MOCK_REGISTRY_FILEPATH, TEMPORARY_DOMAIN)
+from nucypher.utilities.sandbox.middleware import MockRestMiddleware
+from nucypher.utilities.sandbox.ursula import start_pytest_ursula_services
+from web3 import Web3
+
+STAKE_VALUE = MIN_ALLOWED_LOCKED * 2
+POLICY_RATE = Web3.toWei(21, 'gwei')
+POLICY_VALUE = POLICY_RATE * MIN_LOCKED_PERIODS   # * len(ursula)
+
+
+@pytest.fixture(autouse=True, scope='module')
+def funded_blockchain(deployed_blockchain):
+
+    # Who are ya'?
+    blockchain, _deployer_address, registry = deployed_blockchain
+    deployer_address, *everyone_else, staking_participant = blockchain.interface.w3.eth.accounts
+
+    # Free ETH!!!
+    blockchain.ether_airdrop(amount=1000000)
```

Comment:
This needs something similar to `Web3.toWei` here. In general, every expression involving ETH and NU literals should use that approach.

Reply:
Yeah - this appears in several locations all around the codebase.  So far - we have only used the smallest denominations.  I think a better solution can be implemented in a future PR.

Reply:
Following up in PR #830